.eslintrc.js

```diff
@@ -81,14 +81,6 @@ const APACHE_2_0_LICENSE_HEADER = `
  */
 `;
 
-const ELASTIC_LICENSE_HEADER = `
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
For some reason I thought we had to keep this when generating the notice files. But Im down to remove it!